README.md

```diff
@@ -11,31 +11,40 @@ want to print the results and that purpose anonymous object serves just fine.
 
 ### Task
 All transactions within the shops are known: what, when, where and for how much was an item baught.
+
+Create a command line program that accepts the following arguments, 
+
+`programname "path/to/input" "command" "path/to/output"`. The "command" format will be specified in the requirements below (`in parantheses`).
```

Comment:
Leaves no ambiguity as to arguments required. Went ahead and wrapped everything in quotes because tests are invoked by calling main (passing an array of strings).

Reply:
Yes, makes sense.